azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/LifecycleAction.kt

```diff
@@ -4,6 +4,9 @@
 package com.azure.android.communication.ui.chat.redux.action
 
 internal sealed class LifecycleAction : Action {
+    class Wakeup: LifecycleAction()
+    class Shutdown: LifecycleAction()
```

Comment:
Adam, we and Shauna are working on chat initialization this iteration. 
We will look at this code and will get some thoughts and will sync with iOS team. 
As per discussion, we will send PR.